package.json

```diff
@@ -150,7 +150,7 @@
     "prettier-check": "prettier --check \"**/*\""
   },
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=16.0.0"
```

Comment:
[Double checking this now](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines), I believe NodeJS is required for _development_ on this package (running the build and other dev commands with the `--workspaces` flag) and does belong in the build guide, but not for apps which depend on this package. I think we can safely restore this to `14` for those installing the package on npm.

Reply:
Updated.